packages/launchdarkly/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "@sentry/launchdarkly",
+  "version": "8.35.0",
+  "description": "Sentry SDK integration for Launch Darkly feature flagging",
+  "repository": "git://github.com/getsentry/sentry-javascript.git",
+  "homepage": "https://github.com/getsentry/sentry-javascript/tree/master/packages/launchdarkly",
+  "author": "Sentry",
+  "license": "MIT",
+  "engines": {
+    "node": ">=14.18"
+  },
+  "files": [
+    "/build/npm"
+  ],
+  "main": "build/npm/cjs/index.js",
+  "module": "build/npm/esm/index.js",
+  "types": "build/npm/types/index.d.ts",
+  "exports": {
+    "./package.json": "./package.json",
+    ".": {
+      "import": {
+        "types": "./build/npm/types/index.d.ts",
+        "default": "./build/npm/esm/index.js"
+      },
+      "require": {
+        "types": "./build/npm/types/index.d.ts",
+        "default": "./build/npm/cjs/index.js"
+      }
+    }
+  },
+  "typesVersions": {
+    "<4.9": {
+      "build/npm/types/index.d.ts": [
+        "build/npm/types-ts3.8/index.d.ts"
+      ]
+    }
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@sentry/browser": "^8.35.0",
+    "@sentry/core": "8.35.0",
+    "@sentry/types": "*",
+    "@sentry/utils": "*",
```

Comment:
I think these versions should all be the same as current repo version (8.37.1) -- make sure you run `yarn` after you update

Reply:
Can/should I add a `^` to any of these?